src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -306,6 +316,28 @@ public Path saveMediaToWorkspace(MediaEntity mediaEntity) throws IOException {
         return downloadPath;
     }
 
+    private boolean checkAllMediaExistsInTwoStorageLocations(List<MediaEntity> mediaEntities,
```

Comment:
minor: 
pity we don't have an `ExternalLocationTypeService`, this method would have been a good candidate for a service. since I think this method could be useful in other places to in the future. And also would have reduced the complexity of `AudioTransformationServiceImpl` which is already quite complex.
We could consider introducing an `ExternalLocationTypeService` if you like